flytekit/types/file/file.py

```diff
@@ -179,13 +179,21 @@ def extension(cls) -> str:
         return ""
 
     @classmethod
-    def new_remote_file(cls, name: typing.Optional[str] = None) -> FlyteFile:
+    def new_remote_file(cls, alt: typing.Optional[str] = None, name: typing.Optional[str] = None) -> FlyteFile:
```

Comment:
Can you add `alt` as the last parameter?

Reply:
Why is that, out of curiosity? I put it ahead to match it's location in the final path.

Reply:
This is just to maintain backwards compatibility.

Reply:
DUH. I'm so deep in named arguments now that I forgot about positional ones 🤦